packages/vkui/src/components/Snackbar/subcomponents/Basic/Basic.tsx

```diff
@@ -69,6 +76,7 @@ export function Basic({
         styles['Snackbar__body'],
         stylesLayout[layout],
         mode === 'dark' && styles['Snackbar--mode-dark'],
+        sizeY && sizeY !== 'compact' && sizeYClassNames[sizeY],
```

Comment:
```suggestion
        sizeY !== 'compact' && sizeYClassNames[sizeY],
```

с учётом `sizeY = 'none'`